app/controllers/comments_controller.rb

```diff
@@ -16,4 +16,12 @@ def create
       render :new
     end
   end
+
+  def destroy
+    @user = @current_user
+    @post = Post.find_by(id: params[:post_id])
+    @comment = Comment.find_by(id: params[:id])
+    @comment.destroy
+    redirect_to user_post_comments_path(@user, @post), notice: 'Comment successfully deleted.'
+  end
```

Comment:
- [ ]  Great job for creating a destroy method for the comment controller. However, you should also create a destroy method for the post controller. This method will be used when the user is authorized to delete a post and will invoke the destroy method to delete the post.